### PR TITLE
Add release_report_cover_image to Version

### LINKED ISSUE #1467: Add placeholder image to report cover
we would like the below image on the cover. It would be best if the image could have a name that would match the release, like "1-87.png" or something like that, so it could be placed or loaded easily in the future.

Image

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1467
- add an ImageField to the Version model to be used in the release report

### CODE REVIEW

**Comment by brianjp93** [comment]
image

**Review by rbbeeston — COMMENTED** [review]
looks good, can you add some documentation to the ticket on how to go about updating a pic?

**Review by GregKaleka — APPROVED** [review]
LGTM - one question about small screen sizes.

File: templates/admin/release_report_detail.html [hunk 1]
```diff
@@ -55,7 +55,15 @@ <h1>Boost</h1>
           </table>
           {% endif %}
         </div>
-        <div class="flex flex-col">
+        <div class="flex flex-col h-full gap-y-4">
```
[the remark this answers is not in the document]
Comment (brianjp93) [reply templates/admin/release_report_detail.html:58]: This is intended to be viewed only to be saved as a pdf, I don't think mobile is a consideration in this case.  This is also not meant to be the final styling of the page, I want to hand it off to Rob for that.  Maybe it's fine to ignore any styling concerns for now.

**Comment by rbbeeston** [comment]
Yes, just for PDF, don't let it hold up approval

File: templates/admin/release_report_detail.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (GregKaleka) [reply templates/admin/release_report_detail.html:58]: lolz yep never mind. Missed that this was PDF only.

**Pull request merged**